### ISSUE TITLE
Choose a License

### ISSUE THREAD

**Issue by gaurav21r** [rcorp/standard-project-structure#1]
MIT / Apache / BSD ?

This is an Open Source Project for us AND the community and we need to be as liberal as possible with License.

**Comment by riteshe63** [comment]
@gaurav21r 
Choose MIT Licence. 

Because most top github project prefer MIT Licence like:

https://github.com/twbs/bootstrap/blob/master/LICENSE
https://github.com/impress/impress.js/blob/master/LICENSE
https://github.com/angular/angular/blob/master/LICENSE
https://github.com/angular/angular/blob/master/LICENSE
https://github.com/feross/standard/blob/master/LICENSE

**Comment by gaurav21r** [comment]
@riteshe63 Yes MIT Looks best, but I want to see the License chosen for Open Source Projects with Enterpise Consulting. As well as parts of the system which can be expanded Closed Source. Apache is popular with Google. 

I'll see Android / Chrome, Loopback, Dojo 2.0 etc and arrive at a decision.

**Comment by gaurav21r** [comment]
I think the Apache License looks best. Since our organisation @rcorp is involved, I've seen consultants and large technology companies using this license.

From http://choosealicense.com/licenses/  The following points are additional in Apache as compared to MIT:
- Explicitly states that it does NOT grant you trademark rights.
- Indicate changes made to Code.
- Explicit PATENT grant.

Remember we are saying THIS project is Apache Licensed. We are not recommending projects that USE this structure should have this License. We are only saying Projects that fork & modify this code (and maybe contribute back) should follow this License.

**Comment by gaurav21r** [comment]
@riteshe63 Please add a LICENSE file and Include a copyright notice in README.md and close this issue!

**Comment by riteshe63** [comment]
Ok sir i am close this issue.
:+1:

**Comment by gaurav21r** [comment]
@riteshe63 No license info in the README!
